### PR TITLE
Update Release Doc

### PR DESCRIPTION
**Pull request by scottilee**
Adds more detail to the release doc and removes a redundant release file.

/assign @palnabarun

### CODE REVIEW

File: devel/release.md [hunk 1]
```diff
@@ -1,8 +1,71 @@
...
+[this list](https://pypi.python.org/pypi?%3Aaction=list_classifiers).
+
+After changing constants to proper versions, update the client using this
+command:
+
+```bash
+scripts/update-client.sh
+```
+
+Remove the tests directory (ref: https://github.com/kubernetes-client/python/commit/ec9c944f076999543cd2122aff2d86f969d82548).
```
Comment (roycaihw) [inline devel/release.md:58]: Could you move this line to the "hot issues" section, and link to the upstream issue https://github.com/OpenAPITools/openapi-generator/issues/5377?

**Review by roycaihw — COMMENTED** [review]
One comment. LGTM otherwise

**Review by palnabarun — CHANGES_REQUESTED** [review]
Looks good to me other than what @roycaihw mentioned.

**Comment by roycaihw** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**